.gofmt_exclude

```diff
@@ -0,0 +1,7 @@
+bindata_assetfs\.go
```

Comment:
@aslakknutsen I thought so too but actually that file is broken and I have a fix here: https://github.com/fabric8-services/fabric8-wit/pull/1492